packages/headless/src/controllers/facets/facet/headless-facet-options.ts

```diff
@@ -12,10 +12,12 @@ import {
   facetSearch,
 } from '../_common/facet-option-definitions';
 
-export type FacetOptions = Omit<FacetRegistrationOptions, 'facetId'> & {
-  facetId?: string;
-  facetSearch?: Partial<FacetSearchRequestOptions>;
-};
+type OptionalFacetId = Partial<Pick<FacetRegistrationOptions, 'facetId'>>;
```

Comment:
Deriving from the core interface instead of retyping displays the jsdoc.

Reply:
Nice :)